kafka/client_async.py

```diff
@@ -419,10 +424,12 @@ def close(self, node_id=None):
         with self._lock:
             if node_id is None:
```

Comment:
while the code is relatively clear, it'd be nice if the docstring stated that `node_id=None` results in closing all broker connections. The docstring implies it, but doesn't say that explicitly.